starknet_py/tests/e2e/client/conftest.py

```diff
@@ -101,13 +114,15 @@ def fixture_invoke_transaction_selector(invoke_transaction):
 
 
 @pytest.fixture(name="deploy_transaction_hash")
-def fixture_deploy_transaction_hash():
-    return 0x11C1C6731ACE34AB4A9137A82092F26ECE38E7428E5E2028DA587893AAE0E02
+def fixture_deploy_transaction_hash(run_prepared_devnet):
+    _, prepared_data = run_prepared_devnet
+    return int(prepared_data["deploy_transaction_hash"], 16)
```

Comment:
we could convert all of those into the destination type in the previous fixture for ease of use